eng/common/pipelines/templates/jobs/npm-release-task.yml

```diff
@@ -0,0 +1,158 @@
+parameters:
+  Artifact: {}
+  Registry: ''
+  PathToArtifacts: ''
+  DependsOn: []
+  VarDependencies: {}
+  Environment: 'package-publish'
+  TopLevelArtifactName: 'packages'
+
+jobs:
+- ${{ if eq(parameters.Registry, 'https://registry.npmjs.org/') }}:
+  - deployment: PublishPackage_${{ replace(parameters.Artifact.name, '-', '_') }}
```

Comment:
```suggestion
  - deployment: PublishPackage_${{ parameters.Artifact.name }}
```

Reply:
note: just pass in the safe package name